mobile/views/TabBar/index.js

```diff
@@ -17,6 +17,19 @@ import {
   ProfileIcon,
 } from './style';
 
+const tabBarVisible = navigation => {
+  const { routes } = navigation.state;
+  const nonTabbarRoutes = ['Thread', 'ThreadDetail', 'DirectMessageThread'];
```

Comment:
Instead of hardcoding a list of strings here can we use the `hideTabbar` (might be slighlty differently named) option on the individual screens? This is a bit smelly, as this code is somewhere entirely different than the screen definitions so it'll be very unobvious for future debuggers to find.

Reply:
We cannot, unfortunately - this api changed with `react-navigation` 2.0 and we can't do it on individual screens; this is the best solution I could find :/

Reply:
Ugh that sucks. :confused: Alright then